caddyconfig/httpcaddyfile/builtins.go

```diff
@@ -915,7 +915,7 @@ func parseLogHelper(h Helper, globalLogNames map[string]struct{}) ([]ConfigValue
 	// this is useful for setting up loggers per subdomain in a site block
 	// with a wildcard domain
 	customHostnames := []string{}
-
+	noLoggerNames := false
```

Comment:
I guess this variable name is misleading since we chose `no_hostname` as the config option